system/services/routeHandlers/AssetRouteHandler.cfc

```diff
@@ -49,10 +55,16 @@ component implements="iRouteHandler" singleton=true presideService=true {
 		var trackDownload = buildArgs.trackDownload ?: false;
 		var trashed       = IsBoolean( buildArgs.trashed ?: "" ) && buildArgs.trashed;
 		var link          = "";
-		var assetType     = _getAssetManagerService().getAsset(
-			  id           = assetId
-			, selectFields = [ "asset_type" ]
-		).asset_type ?: "";
+		var tenantField   = Trim( _getTenancyService().getObjectTenant( "asset" ) );
```

Comment:
This isn't necessarily the tenant field - for that you need `getTenantFkForObject()`. Also I suggest it would make sense to cache this as a variable in this singleton (it won't change in the duration of the application lifetime). So, a `_getAssetTenancyField()` helper, that looks something like:

```cfc
if ( !StructKeyExists( variables, "_assetTenantField" ) {
    variables._assetTenantField = _getTenancyService().getTenantFkForObject( "asset" );
}
return variables._assetTenantField;
```